src/index.html

```diff
@@ -84,7 +84,7 @@ <h2 class="h3">{% t index-activity.title %}</h2>
 
 
 <!-- Partner panel -->
-<div class="section gray-section">
+<section class="section gray-section">
```

Comment:
Guessing we're going to do more with these classes to make them more descriptive, and then remove the comments? Looks great so far though!

Reply:
Yeah exactly. Once they follow the BEM convention they'll already be descriptive and the comments would then be redundant